src/tasks/BagelAITools.py

```
@@ -152,6 +152,17 @@ def BASE_BAGEL_I18N(self):
             self.log_error(f"Error loading i18n from {json_path}", e)
         return _i18n_cache
 
+    @cached_property
+    def def_prompt(self):
+        try:
+            _prompt_cache = []
+            for lang_data in self.BASE_BAGEL_I18N["model_prompt"].values():
+                _prompt_cache.extend(lang_data.values())
+        except Exception as e:
+            self.log_error(f"Error loading default prompts ", e)
```

Ruff (0.15.15)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)